specification/servicefabric/resource-manager/Microsoft.ServiceFabric/stable/2020-03-01/application.json

```diff
@@ -1012,6 +1012,10 @@
     "ApplicationResource": {
       "description": "The application resource.",
       "properties": {
+        "identity": {
+          "$ref": "#/definitions/ManagedIdentity",
+          "description": "Describes the managed identities for an Azure resource."
```

Comment:
I'm a little bit confused. Why is it `managed indentities` in `identity` field? There is another field called `managedIdentities`